Tools/AP_Periph/esc_apd_telem.cpp

```diff
@@ -0,0 +1,103 @@
+/*
+  ESC Telemetry for Hobbywing Pro 80A HV ESC. This will be
+  incorporated into a broader ESC telemetry library in ArduPilot
```

Comment:
... right.... sure it will ;-)

Reply:
HWESC was the template, I don't care enough about it though, so removed.